include/ultra64/gbi.h

```diff
@@ -1,4 +1,5 @@
 #include "mbi.h"
+#include "ultratypes.h"
```

Comment:
This should be able to go under the header guard

Reply:
oh I didnt even realize it was above it. I just put it under mbi.h which was already outside.
Can both go under?

Reply:
seems yes, it built fine